yb-voyager/src/query/queryissue/parser_issue_detector_test.go

```diff
@@ -486,6 +486,130 @@ func TestSingleXMLIssueIsDetected(t *testing.T) {
 	assert.Equal(t, 1, len(issues))
 }
 
+func TestJsonUnsupportedFeatures(t *testing.T) {
```

Comment:
There is a difference in way how we get queries from pgg(with `$1, $2`) vs how we are testing here(with values)

We need to be consistent in all tests soon, we might miss some bug because of this.

Reply:
Yes we should add unit tests with mix of both.